contracts/testHelpers/ChainId.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.7.3;
+
+import "../common/MultiChain.sol";
+
+contract ChainId is MultiChain {
```

Comment:
You know, we could probably just drop this helper entirely and use `MultiChain` in the tests directly.